javatests/arcs/core/host/PausableTestingHost.kt

```diff
@@ -0,0 +1,32 @@
+package arcs.core.host
+
+import arcs.core.storage.testutil.testStorageEndpointManager
+import arcs.jvm.util.testutil.FakeTime
+
+/** Fake [ArcHost] that let's us view pausing behavior for tests. */
+class PausableTestingHost(
+  schedulerProvider: SchedulerProvider,
+  vararg particles: ParticleRegistration
+) : TestingHost(
```

Comment:
`TestingHost` and its superclass `AbstractArcHost` do a _lot_ of stuff.

I'd recommend implementing a `NoopArcHost` which implements the `ArcHost` interface directly, and returns Unit, or empty values, or throws, depending on what kind of return type is expected for each method.

Then this `PausableTestingHost` (which I would rename to `FakeArcHost` and put directly inside the unit test file where it's used) becomes much simpler, since it just has to implement the `pause` and `unpause` methods.

It makes all of this much easier to reason about, since you know the base class doesn't actually do anything.